python/ray/util/sgd/v2/worker_group.py

```diff
@@ -43,6 +56,11 @@ class WorkerGroup:
             Dictionary specifying the extra resources that will be
             requested for each worker in addition to ``num_cpus_per_worker``
             and ``num_gpus_per_worker``.
+        actor_cls (Optional[Type]): If specified use this class as the
+            remote actors. The provided class is expected to extend
+            ``BaseWorker``.
```

Comment:
No need for second line anymore since it's performed directly in `__init__`.

Reply:
Fixed!